mayastor/src/bdev/nexus/nexus_channel.rs

```diff
@@ -26,8 +26,8 @@ pub(crate) struct NexusChannel {
 #[repr(C)]
 #[derive(Debug)]
 pub(crate) struct NexusChannelInner {
-    pub(crate) ch: Vec<BdevHandle>,
-    pub(crate) write_only: usize,
+    pub(crate) write_ch: Vec<BdevHandle>,
```

Comment:
write_only was a poor name (my bad) so I wonder if write_ch should just be name something else, since it's actually used for different operations and not just write. Maybe even just ch?

Reply:
I think `write_ch` is fine, it's sometimes used in certain contexts to mean any modifying operation.

Reply:
that's true, although in this case it's not that we have a read and a write handle which are opened differently. The read/write here is simply to indicate that a child is not up to date so cannot be read from as it might return invalid user data.
I wonder actually, when we take snapshots, maybe that should be done only on the read_ch?

Reply:
I suppose creating a snapshot should be done on `read_ch` though `nexus_bdev::Nexus::nvme_admin`, as written, applies to any NVMe Admin command. You could select the channel based on the opcode, or just leave a fixme comment behind until we get round to addressing snapshots.